package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@evojam/redux-utils",
+  "version": "1.0.0",
+  "main": "./src",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": "https://github.com/evojam/redux-utils.git",
+  "homepage": "https://github.com/evojam/redux-utils.git#readme",
+  "bugs": {
+    "url": "https://github.com/evojam/redux-utils/issues"
+  },
+  "author": "Krzysztof Antecki <kantecki@evojam.com>",
+  "license": "MIT",
+  "peerDependencies": {
+    "typescript": ">=2.8"
+  },
+  "devDependencies": {
+    "@types/jest": "^23.3.2",
+    "jest": "^23.6.0",
+    "ts-jest": "^23.10.0",
+    "typescript": "^3.0.3"
```

Comment:
Here you have 3.0.3 but you forced peerdependency of `<3.0` as minimal requirement.

So if you use any of 3.0 features then all projects which use `<3.0` will break even if it matches `peerDependency` requirement.

You should set there `>= 2.8` and `<3.0`

Reply:
It's necessary to have TS `>=2.8` as pointed in `peerDependencies` so I'll make them equal. 
I guess I'm not using anything of 3.0 or above features 🤔